src/components/pages/weather/elements/WeatherTempCell.js

```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react';
 
 export default class WeatherTempCell extends Component {
-  render() {
-    const averageTemp = this.props.dayTemperatureData.avg.toFixed(3);
-    const maxTemp = this.props.dayTemperatureData.max.toFixed(3);
-    const minTemp = this.props.dayTemperatureData.min.toFixed(3);
+  constructor(props) {
+    super(props);
+    this.temperatureCellClass = null;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getTemperatureCellClass(averageTemp) {
```

Comment:
The error is trying to tell you that this should be a static method as it doesn't use any of the properties of the individual element.

The alternative is to have it take no arguments and use `this.props.dayTemperature.avg` instead.

Up to you!